docs/getting-started/supported-languages-and-tools.md

```diff
@@ -82,7 +82,9 @@ The table below lists all programming languages currently supported by Codacy an
     <tr>
       <td>Go</td>
       <td><a href="https://github.com/securego/gosec">Gosec</a><a href="#client-side"><sup>*</sup></a>,
-          <a href="https://github.com/mgechev/revive">Revive</a></td>
+          <a href="https://github.com/mgechev/revive">Revive</a>,
+          <a href="https://staticcheck.io/docs">Staticcheck</a><a href="#client-side"><sup>*</sup></a>
```

Comment:
For consistency, I suggest that we link directly to the home page of the project instead of the docs:

```suggestion
          <a href="https://staticcheck.io/">Staticcheck</a><a href="#client-side"><sup>*</sup></a>
```

The home page of the tools normally does a better job of explaining what the tool is and quickly listing the supported languages / rules. From there, our users should be able to easily navigate to the documentation if they want to know more.